### ISSUE TITLE
NPM 0.3.2 install failure

### ISSUE THREAD

**Issue by micahriggan** [krzkaczor/TypeChain#90]
When running
```
npm install --save-dev typechain
```
I get this
```
npm ERR! path myproj/node_modules/typechain/dist/cli.js
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod 'myproj/node_modules/typechain/dist/cli.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent
```
I think maybe cli.js didn't get included in the npm publish

**Comment by vkantchev** [comment]
Also present in 0.3.1. The latest version that installs fine is 0.2.7

**Comment by krzkaczor** [comment]
@micahriggan @vkantchev thanks for the report. I will check it asap.

**Comment by krzkaczor** [comment]
Fix released as `typechain@0.3.3`. Sorry for that. The upside is that I added tests that will verify it for future.

**Comment by micahriggan** [comment]
No worries! Thanks for making this
